test/integration/driver_install_or_update_test.go

```diff
@@ -173,3 +173,8 @@ func TestHyperKitDriverInstallOrUpdate(t *testing.T) {
 		}
 	}
 }
+
+func sudoNeedsPassword() bool {
+	err := exec.Command("sudo", "-n", "ls").Run()
```

Comment:
this wont work on windows, I recommend skipping this test for non-darwin

Reply:
This function is used only in tests related to hyperkit driver. They don't run on windows.